### DISCUSSION TITLE
Feature request: jump to start or end of a `let .. in` block

### DISCUSSION THREAD

**Discussion by mimoo** [ocamllabs/vscode-ocaml-platform#1171] [Q&A]
copying from https://stackoverflow.com/questions/75229023/how-do-i-jump-to-the-start-or-the-end-of-an-ocaml-block-in-vscode/75229168#75229168

Sometimes I look at code that goes like this:

```ocaml
let x =   
  (* 1 million lines of code *)
  let y =
     (* 5 billion lines of code *)
  in
  (* 5 lines of code *)
in
```

and I find myself scrolling with my cursor on the right indentation to find where a `let ... in` starts or ends. Is there a better way to do this in VSCode?

**Comment by lukstafi** [comment]
I think this is not sufficiently `vscode-ocaml-platform` specific as you can use language-agnostic tools to this effect. I'm biased, but: [Navi Parens](https://marketplace.visualstudio.com/items?itemName=lukstafi.navi-parens).
